javascript/javascript2/week1/spirit_animal/spiritAnimalGenerator.js

```diff
@@ -0,0 +1,31 @@
+const spiritAnimalName = ['Strong and Courageous Bear', 'The incredible butterfly',
+    'The Patience Cat', 'The Innocent Deer',
+    'The Friendly Dolphin', 'The Cunning Fox', 'The black Panter',
+    'A German Shepherd', 'The Roar of a Lion', 'A well visionery Hawk'];
```

Comment:
Arrays like this are best formatted with one item on each line.
```suggestion
const spiritAnimalName = [
    'Strong and Courageous Bear',
    'The incredible butterfly',
    'The Patience Cat',
    'The Innocent Deer',
    'The Friendly Dolphin',
    'The Cunning Fox',
    'The black Panter',
    'A German Shepherd',
    'The Roar of a Lion',
    'A well visionery Hawk',
];
```